docs/sources/whatsnew/whats-new-in-v10-0.md

```diff
@@ -0,0 +1,277 @@
+
+### Public dashboards insights
+
+_Available in preview in Grafana Enterprise, Cloud Pro, and Cloud Advanced._
+
+The public dashboards insights provide valuable information about your public dashboard usage. You can easily access and view important metrics such as the daily query count, the number of views in the last 30 days, and the number of errors in the last 30 days.
+
+To try it out, you'll need to enable the `publicDashboards` feature toggle. If you’re using Grafana Cloud, and would like to enable this feature, please contact customer support.
+
+### Email sharing for public dashboards
+
+_Available in preview in Grafana Enterprise, Cloud Pro, and Cloud Advanced._
+
+Our email sharing feature allows you to easily share your public dashboards with specific individuals. By adding their email addresses, they will receive a one-time valid link to access the dashboard. This provides you with greater control over who can visualize your data.
+
+To try it out, you'll need to enable the `publicDashboardsEmailSharing` and `publicDashboards` feature toggles. If you’re using Grafana Cloud, and would like to enable this feature, please contact customer support.
+
+## Scenes
+
+_This is an experiemental library_
+
+Scenes is a new front-end library by Grafana that empowers application engineers to effortlessly build stunning dashboard experiences right into their products. With Scenes, you can easily create apps that mirror the Grafana dashboarding experience, complete with template variable support, flexible layouts, dynamic panel rendering, and so much more.
+
+To try it out, please check [@grafana/scenes](https://github.com/grafana/scenes).
+
+<!--
+- TODO: Add link to docs and resources
+-->
+
+## Query multiple data sources in Explore
+
+_Available in preview in all editions of Grafana using the exploreMixedDatasource feature toggle in all editions of Grafana._
+
+You can now query multiple data sources in Explore. Select "Mixed" from the data source picker and specify a data source for each query.
+
+Mixed data source in Explore is gradually rolling out to all users on Grafana Cloud. If you’re using Grafana Open Source and Enterprise, you can enable this feature using the exploreMixedDatasource feature toggle.
+
+## Span filtering for traces
+
+_Available in preview in all editions of Grafana._
+
+You can now work much more efficiently with traces that consist of a large number of spans.
+
+The span filters exist above the trace view, and allow you to filter the spans that are shown in the trace view. The more filters you add, the more specific are the filtered spans.
+
+Currently, you can add one or more of the following filters:
+
+- Service name,
+- Span name,
+- Duration,
+- Tags (which include tags, process tags, and log fields).
+
+To try it out, you'll need to enable the `newTraceView` feature toggle. If you’re using Grafana Cloud, and would like to enable this feature, please contact customer support.
+
+## Datagrid
+
```

Comment:
@oscarkilhed - I want to nest your items (Datagrid and Drag and drop) under the Dashboards and visualizations heading that I've added. Any objections?

Reply:
@imatwawana - sounds good (@oscarkilhed is out)